test-data/unit/python2eval.test

```diff
@@ -128,6 +128,21 @@ print '>', ['a', b'b', u'c']
 [out]
 > [u'a', 'b', u'c']
 
+[case testUnicodeLiteralsKwargs_python2]
```

Comment:
Can you also add a test that doesn't import unicode_literals but uses u'...' literals?